app/models/tag.rb

```diff
@@ -0,0 +1,3 @@
+class Tag < ApplicationRecord
+  has_many :yardsale_tags
+end
```

Comment:
@djimenez017 What do you think about using this gem instead? https://github.com/mbleigh/acts-as-taggable-on

Reply:
i am not opposed, it seems like it would come in handy when filtering by tags

Reply:
@djimenez017 yeah or you could do it on a separate PR